### DISCUSSION TITLE
Does Termux Pacman packages has all packages?

### DISCUSSION THREAD

**Discussion by ghost** [termux/termux-packages#17544] [Q&A] [answered]
Does Termux pacman has all packages which termux packages has? (Excluded apt dpkg etc)

**Comment by Maxython** [comment, marked as answer]
**In short:** yes.

**In more detail:** yes, but there is one point, @termux-pacman is more serious about packages than @termux. **For example:** if a package is disabled or removed from the GitHub repo, it will remain in the @termux service. But in the @termux-pacman service, it will be removed.

**Reply by ghost** [reply]
I have a question, i want a package but I don't know how can I modify it to work on termux, it has source code on github, and package I want is plank dock.
